### PR TITLE
🌐 Add Portuguese translation for `docs/pt/docs/tutorial/background-tasks.md`

### PR DESCRIPTION
**Pull request by izaguerreiro**

### CODE REVIEW

**Review by rsip22 — APPROVED** [review]
Great work! 👍🏻

File: docs/pt/docs/tutorial/background-tasks.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+# Tarefas em segundo plano
+
+Você pode definir tarefas em segundo plano a serem executadas _ após _ retornar uma resposta.
+
+Isso é útil para operações que precisam acontecer após uma solicitação, mas que o cliente realmente não precisa estar esperando a operação ser concluída para receber a resposta.
```
Comment (LorhanSohaky) [inline docs/pt/docs/tutorial/background-tasks.md:5]:
```suggestion
Isso é útil para operações que precisam acontecer após uma solicitação, mas que o cliente realmente não precisa esperar a operação ser concluída para receber a resposta.
```

File: docs/pt/docs/tutorial/background-tasks.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+Isso inclui, por exemplo:
+
+- Notificações por email enviadas após a realização de uma ação:
```
Comment (LorhanSohaky) [inline docs/pt/docs/tutorial/background-tasks.md:9]:
```suggestion
- Envio de notificações por email após a realização de uma ação:
```

File: docs/pt/docs/tutorial/background-tasks.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+## Usando `BackgroundTasks`
+
+Primeiro, importe `BackgroundTasks` e defina um parâmetro em sua _função de operação de caminho_ com uma declaração de tipo de `BackgroundTasks`:
+
+```Python hl_lines="1  13"
+{!../../../docs_src/background_tasks/tutorial001.py!}
+```
+
+**FastAPI** criará o objeto do tipo `BackgroundTasks` para você e o passará como esse parâmetro.
```
Comment (LorhanSohaky) [inline docs/pt/docs/tutorial/background-tasks.md:22]:
```suggestion
O **FastAPI** criará o objeto do tipo `BackgroundTasks` para você e o passará como esse parâmetro.
```

File: docs/pt/docs/tutorial/background-tasks.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+
+## Criar uma função de tarefa
+
+Crie uma função a ser executada como tarefa em segundo plano.
+
+É apenas uma função padrão que pode receber parâmetros.
+
+Pode ser uma função `async def` ou `def` normal, **FastAPI** saberá como lidar com isso corretamente.
```
Comment (LorhanSohaky) [inline docs/pt/docs/tutorial/background-tasks.md:30]:
```suggestion
Pode ser uma função `async def` ou `def` normal, o **FastAPI** saberá como lidar com isso corretamente.
```

File: docs/pt/docs/tutorial/background-tasks.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+
+- Uma função de tarefa a ser executada em segundo plano (`write_notification`).
+- Qualquer sequência de argumentos que deve ser passada para a função de tarefa na ordem (`email`).
+- Quaisquer argumentos de palavra-chave que devem ser passados ​​para a função de tarefa (`mensagem = "alguma notificação"`).
+
+## Injeção de dependência
+
+Usar `BackgroundTasks` também funciona com o sistema de injeção de dependência, você pode declarar um parâmetro do tipo `BackgroundTasks` em vários níveis: em uma _função de operação de caminho_, em uma dependência (confiável), em uma subdependência, etc.
+
+**FastAPI** sabe o que fazer em cada caso e como reutilizar o mesmo objeto, de forma que todas as tarefas em segundo plano sejam mescladas e executadas em segundo plano posteriormente:
```
Comment (LorhanSohaky) [inline docs/pt/docs/tutorial/background-tasks.md:58]:
```suggestion
O **FastAPI** sabe o que fazer em cada caso e como reutilizar o mesmo objeto, de forma que todas as tarefas em segundo plano sejam mescladas e executadas em segundo plano posteriormente:
```

File: docs/pt/docs/tutorial/background-tasks.md [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+
+Se houver uma consulta na solicitação, ela será gravada no log em uma tarefa em segundo plano.
+
+E então outra tarefa em segundo plano gerada na _função de operação de caminho_ escreverá uma mensagem usando o parâmetro de caminho `email`.
+
+## Detalhes técnicos
+
+A classe `BackgroundTasks` vem diretamente de <a href="https://www.starlette.io/background/" class="external-link" target="_blank">`starlette.background`</a>.
+
+Ele é importado/incluído diretamente no FastAPI para que você possa importá-lo do `fastapi` e evitar a importação acidental da alternativa `BackgroundTask` (sem o `s` no final) de `starlette.background`.
```
Comment (LorhanSohaky) [inline docs/pt/docs/tutorial/background-tasks.md:74]:
```suggestion
Ela é importada/incluída diretamente no FastAPI para que você possa importá-la do `fastapi` e evitar a importação acidental da alternativa `BackgroundTask` (sem o `s` no final) de `starlette.background`.
```

File: docs/pt/docs/tutorial/background-tasks.md [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+
+Usando apenas `BackgroundTasks` (e não `BackgroundTask`), é então possível usá-lo como um parâmetro de _função de operação de caminho_ e fazer **FastAPI** cuidar do resto para você, assim como ao usar o objeto `Request` diretamente.
```
Comment (LorhanSohaky) [inline docs/pt/docs/tutorial/background-tasks.md:76]:
```suggestion
Usando apenas `BackgroundTasks` (e não `BackgroundTask`), é então possível usá-la como um parâmetro de _função de operação de caminho_ e deixar o **FastAPI** cuidar do resto para você, assim como ao usar o objeto `Request` diretamente.
```

File: docs/pt/docs/tutorial/background-tasks.md [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+
+Ainda é possível usar `BackgroundTask` sozinho no FastAPI, mas você deve criar o objeto em seu código e retornar uma Starlette `Response` incluindo-o.
+
+Você pode ver mais detalhes nos <a href="https://www.starlette.io/background/" class="external-link" target="_blank"> documentos oficiais da Starlette para tarefas em segundo plano </a>.
```
Comment (LorhanSohaky) [inline docs/pt/docs/tutorial/background-tasks.md:80]:
```suggestion
Você pode ver mais detalhes na <a href="https://www.starlette.io/background/" class="external-link" target="_blank"> documentação oficiais da Starlette para tarefas em segundo plano </a>.
```

**Review by LorhanSohaky — COMMENTED** [review]
I think with these suggestions the text is more natural

**Review by LorhanSohaky — APPROVED** [review]

**Comment by tiangolo** [comment]
Thanks a lot @izaguerreiro! :cake: 

And thanks for the reviews @LorhanSohaky and @rsip22! :cookie:

**Pull request merged**